src/reduced_basis/rb_evaluation.C

```diff
@@ -1054,19 +1061,19 @@ void RBEvaluation::read_in_vectors_from_multiple_files(System & sys,
   Xdr header_data(file_name.str(),
                   read_binary_vectors ? DECODE : READ);
 
-  // set the version number in header_data from io_version_string
-  // (same code as in EquationSystemsIO::_read_impl)
-  std::string io_version_string = get_io_version_string();
-  std::string::size_type lm_pos = io_version_string.find("libMesh");
-  std::istringstream iss(io_version_string.substr(lm_pos + 8));
+  // Set up the header info (based on EquationSystems::_read_impl).
+  std::string version;
+  header_data.data(version);
+  std::string::size_type lm_pos = version.find("libMesh");
```

Comment:
Actually, I suggest a separate patch where we fix both of those at the same time, and not include it in this PR. What do you think?